src/PeerDigest.h

```diff
@@ -81,7 +81,7 @@ class PeerDigest
 
     CbcPointer<CachePeer> peer;           /**< pointer back to peer structure, argh */
     CacheDigest *cd = nullptr;            /**< actual digest structure */
-    SBuf host;                        ///< copy of peer->host
+    const SBuf host;                      ///< copy of peer->host
```

Comment:
Decorations like this are more harmful than useful. Please get rid of the old comment alignment since we have to modify this line:

```suggestion
    const SBuf host; ///< copy of peer->host
```